.github/workflows/git-sync.yml

```diff
@@ -1,6 +1,11 @@
 name: Git Sync
 
-on: push
+on:
+  push:
+    branches-ignore:
+      - beta/*
+      - master
+
```

Comment:
From what I understand:

We're still pushing `"refs/remotes/source/*"` in both cases,
which means that the race condition (2 git sync running at the same time will step on each other toes) still happens,

Shouldn't we push `refs/remote/source/${CURRENT_CI_BRANCH}` so that each CI run takes care of its own branch?

Reply:
Unfortunately that wouldn't solve the problem in case of `master` and `beta/*` branches since those branches will be affected by commits and pushes by `standardci` user. So as you can see we have to have sequential syncs in case of `master` and `beta/*` (after `standardci` changes) and async syncs on any other branch